tests/models/test_zone.py

```diff
@@ -146,10 +147,25 @@ def test_proper_exception_is_raised(self):
         self.assertRaises(SSLUnavailable, self.zone.get_ssl_verification_info)
 
 
-class TestPurgeZone(FakedServiceTestCase):
+class TestPurgeZone(TestCase):
+
     def setUp(self):
-        self.user = User.get(email='foo@example.net')
-        self.zone = self.user.get_zone_by_name('example.com')
+        self.service_mock = patch(
+            'pycloudflare.models.CloudFlareService').start()
+        self.zone = Zone(User('email', 'api_key'), {'id': 'zone_id'})
+
+    def tearDown(self):
+        self.service_mock.stop()
```

Comment:
to `teardown` you have to call `stop` on pather, not mock. also, you can use `self.addCleanup` instead of `tearDown` to stop patching